Form/Extension/CollectionTypeExtension.php

```diff
@@ -27,7 +29,7 @@ class CollectionTypeExtension extends AbstractTypeExtension
      */
     public function getExtendedType()
     {
-        return 'collection';
+        return method_exists(AbstractType::class, 'getBlockPrefix') ? CollectionType::class: 'collection';
```

Comment:
We should have two branch! Not a bundle that check the Symfony version in an `if` statement.